specification/attributes/amortization_handling.md

```diff
@@ -0,0 +1,34 @@
+# Amortization Handling
+
+Amortization is a process used to break down and spread purchase costs over the period of time or term of use. 
+
+Amortization is primarily used in purchasable commitment-based discounts. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on resource usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate significant purchase charges to the appropriate audience in support of their cost allocation efforts. 
```

Comment:
> When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource

I wonder if it's better to say that a purchase is applicable to the usage of a resource rather than to a resource.

Taking example of Azure, reservations can be made for VM usage and for Azure DataBricks (software). So for a single resource, multiple purchases can apply for the same resource: VM reservation and Azure DataBricks reservation.

Reply:
Resolving this since there was a major refactor of the attribute. Can you take another look?

Reply:
resolving as amortization specific information was removed.